backend/src/v1/services/report-calc-service.spec.ts

```diff
@@ -1207,6 +1207,27 @@ describe('calculatePercentReceivingBonusPay', () => {
 });
 
 describe('calculateAll', () => {
+  describe(`when only one gender category has at least ${reportCalcService.MIN_REQUIRED_PEOPLE_COUNT_PER_GENDER} employees`, () => {
+    it(`returns all applicable calculated amounts, but each is suppressed`, async () => {
+      const csvReadable = new Readable();
+      csvReadable.push(
+        `Gender Code,Hours Worked,Ordinary Pay,Special Salary,Overtime Hours,Overtime Pay,Bonus Pay\n`,
+      );
+      Array(reportCalcService.MIN_REQUIRED_PEOPLE_FOR_REPORT).forEach((v) => {
+        csvReadable.push(`${GENDER_CODES.MALE[0]},1,100,0,0,0,0\n`);
+      });
+      csvReadable.push(null);
+      const allCalculatedAmounts: CalculatedAmount[] =
+        await reportCalcService.calculateAll(csvReadable);
+      allCalculatedAmounts.forEach((c) => {
+        if (c.isSuppressed == false) {
+          console.log(c);
```

Comment:
Use logger instead of console

Reply:
oops, the console.log statement should be there at all.  I removed it.